### PR TITLE
Improvements to pages

### PR DESCRIPTION
**Pull request by MN4444**
gave community resources more resources and updated getting started

### CODE REVIEW

**Comment by MN4444** [comment]
i just want to mark this as ready

**Comment by MN4444** [comment]
oh now i understand

File: docs/community.md [hunk 1]
```diff
@@ -3,3 +3,5 @@
 ## Video Tutorials
```
Comment (SquirrelKiev) [inline docs/community.md:3]: This line needs to be changed to make more sense

File: docs/getting-started.md [hunk 2]
```diff
@@ -36,6 +36,7 @@ Create a plane, right click on it's Transform in the Inspector, and click reset.
 Create a Giant spawn position and Warrior spawn position. These can be found in `Assets/Davigo/Common/Prefabs`.
 
 We'll also need a killzone so the warrior respawns when they fall off. This can be found in `Assets/Govidad/Govigedit/Package/Prefabs`.
+!> Don't make the killzone too small as this can lead to problems later down the line
```
Comment (SquirrelKiev) [inline docs/getting-started.md:39]: beloved punctuation

**Review by SquirrelKiev — CHANGES_REQUESTED** [review]

File: docs/community.md (hunk 1 above)
Comment (MN4444) [reply docs/community.md:3]: what do you mean by that

**Review by SquirrelKiev — APPROVED** [review]

**Pull request merged**